cypress/page/shoping-cart.page.ts

```diff
@@ -2,6 +2,7 @@ class ShopingCart {
     private proceedToCheckoutButton: string;
 
     constructor(){
+        // There's no need to improve this selector.
         this.proceedToCheckoutButton =  ".cart_navigation span"
```

Comment:
yes, you can improve it, try to use attributes to indicate we are looking for the proceed to checkout button